packages/cf-component-progress/src/Progress.js

```diff
@@ -1,6 +1,95 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Link from 'cf-component-link';
+import { createComponent } from 'cf-style-container';
+
+const styles = () => ({
+  position: 'relative'
+});
+
+const Bar = createComponent(
+  () => ({
+    display: 'block',
+    width: '100%',
+    appearance: 'none',
+
+    '&::-webkit-progress-bar': {
+      backgroundColor: '#ebebeb',
+      color: '#ebebeb'
+    },
+
+    '&::-webkit-progress-value': {
+      backgroundColor: '#2f7bbf',
+      color: '#2f7bbf',
+      transition: 'width 500ms ease'
+    },
+
+    /* Mozilla uses progress-bar as the value */
+    '&::-moz-progress-bar': {
+      backgroundColor: '#2f7bbf',
```

Comment:
@neynah That section seems to be missing quite a few colors. Are all those added manually?